registry/src/components/packageItem.js

```diff
@@ -45,7 +45,14 @@ const PackageItem = ({ packageEntity }) => {
         </Col>
         <Col md={4}>
           <div>
-            <h5 id="list-item-package-name">{packageEntity.name}</h5>
+            <h5 id="list-item-package-name">
+              <a
+                href={`/packages/${packageEntity.namespace}/${packageEntity.name}`}
```

Comment:
Can you make use of ``Link`` from ``react-router`` instead of ``anchor`` tag ? 

https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function

Reply:
sure.